pkg/apis/kubedirector.bluedata.io/v1alpha1/kubedirectorcluster_types.go

```diff
@@ -105,6 +105,7 @@ type FileInjections struct {
 type Role struct {
 	Name           string                      `json:"id"`
 	Labels         map[string]string           `json:"labels,omitempty"`
```

Comment:
Should we call this PodLabels now?

Reply:
Maybe! Yeah, I did ponder that. It does make sense.